xla/service/gpu/fusions/cudnn_test.cc

```diff
@@ -479,6 +479,32 @@ ENTRY e {
                             ErrorSpec{/*aabs=*/1e-3, /*arel=*/1e-3}));
 }
 
+TEST_F(CuDnnFusionLevel2Test, FuseConstantExecutesCorrectly) {
+  EXPECT_TRUE(RunAndCompare(R"(
```

Comment:
Shouldn't we have both execution tests, and fusion tests? (the latter checking that fusion has actually occurred?)

Reply:
The entire fusion HLO will be serialized into a cudnn graph, and if cudnn can't handle it, cudnn will return a rejection. I didn't get what "the latter checking that fusion has actually occurred" means? Maybe my naming is confusing, changed to ConstantExecutesCorrectly.

Reply:
This PR is adding new fusion capability, right?

You are testing that constructed fusions are executing correctly, but you are not testing that those fusions (`fusion1` in your example) would actually be built?